paymentsheet/src/main/java/com/stripe/android/customersheet/injection/CustomerSheetViewModelModule.kt

```diff
@@ -76,6 +77,13 @@ internal interface CustomerSheetViewModelModule {
             return Dispatchers.IO
         }
 
+        @Provides
+        @PaymentElementCallbackIdentifier
+        fun providesInstanceId(): String {
+            // We don't have shared callback dependencies here
```

Comment:
We kinda do with `CustomerAdapter`.

Reply:
FWIW, I don't think we should put effort into supporting it without a good reason to. So it's totally fine to throw this task out the window. I just think we should be clear in the associated comment.